packages/framework/src/Migrations/Version20191029210140.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20191029210140 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE currencies ADD rounding_type VARCHAR(15) NOT NULL DEFAULT \'integer\'');
+        $this->sql('ALTER TABLE currencies ALTER rounding_type DROP DEFAULT');
+
+        $roundingTypeSetting = $this->sql('SELECT value FROM setting_values WHERE name = \'roundingType\' AND domain_id = 0;')->fetchColumn(0);
+        if ($roundingTypeSetting !== false) {
```

Comment:
I haven't fixed it yet. I prefer working with string over the integer when it comes to types. It really helps easily identify type of record/entity in production. It is really handy when you need to debug something and identify which type of data you are working with.
As I can see in SSFW, there is no standard/rule how to work with such a type of field. Some entities use integer(e.g `OrderStatus`), other use string(e.g. `Advert`).

It will be great if some standard/rule exists for this. 
Maybe it is a time to introduce type `enum` and use this type for such a situations. I know that I discussed it with @pesektomas and guys from implementation team. They are also interested in this.

I will discuss it with @grossmannmartin

Reply:
Why you don't like strings as dials in database, @TomasLudvik ?

Reply:
I, personally, prefer strings over integers for clarity when it comes to working with DB.
As it's a small table I'm not worried about performance or storage size. 

Enum seems like a good idea, but it's harder to extend such fields (you have to add value in your class and create a migration to modify DB column).

Reply:
@pesektomas and @grossmannmartin because strings are much worse for performance optimization in general. You can say, that in this case it will not affect much, but we should be unified in such use, so if we use in this case strings, we should use them always and that might lead in performance problems in future.

Reply:
@TomasLudvik we could performance  solve using indexes, right?

@grossmannmartin You are right with enums, this problem exceeds this user story. But (for inspiration) I like this https://github.com/consistence/consistence-doctrine.

Reply:
@pesektomas strings are slower significantly against integers